examples/simple-form/index.jsx

```diff
@@ -26,24 +26,29 @@ const checkboxProps = {
   values: {},
   options: [
     {
-      value: 'eggs',
+      name: 'eggs',
       label: 'Eggs',
       content: '🐥🐣',
+      checked: true,
+      required: false,
     },
     {
-      value: 'protien',
+      name: 'protien',
```

Comment:
This is 100% irrelevant to the technical accuracy of your PR, and it was already like this before you got to it, but protein is misspelled on lines 36, 37, and 69.

Reply:
thank you, I really need to add a spell checker to vs code 💯

Reply:
I just want to shout out @sdk-irl here for noting that typo. This is a great example of leaving the place better than you found it (one of the lessons drilled into me back when I was doing video production).

Reply:
I really enjoy this one if you are looking for a VS Code extension: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker